src/styles/Select.scss

```
@@ -0,0 +1,4 @@
+.select{
+    border-radius: 5px;
+    margin: 5px;
+}
```

Conventionally you should leave a space between class name and the bracket
```suggestion
.select {
    border-radius: 5px;
    margin: 5px;
}
```